core/lib/dal/src/models/storage_transaction.rs

```diff
@@ -322,6 +322,85 @@ impl From<StorageTransaction> for Transaction {
     }
 }
 
+#[derive(Serialize, Deserialize, sqlx::FromRow)]
+pub struct StorageTransactionReceipt {
```

Comment:
Nit: It could make sense to have `pub(crate)` visibility for this struct; logically, it seems internal to the DAL crate.